lib/src/tasks/coverage/exceptions.dart

```diff
@@ -38,3 +38,10 @@ class MissingLcovException implements Exception {
   final String message = missingLcovMessage;
   String toString() => 'MissingLcovException: $message';
 }
+
+class PortBoundException implements Exception {
+  final String message;
+  PortBoundException(String message)
+      : this.message = 'Port could not be bound: $message';
+  String toString() => 'PortBoundException: $message';
```

Comment:
This exception should include the port number. When the selenium server fails to start (because it's already running), the port number it tried to connect to should be output with the error to make it easy to kill the process.